.dev/compare_branches.R

```diff
@@ -15,14 +17,19 @@
 #     ./dev/compare_branches --branch=master --outfile=old.csv ...
 #   And then compare the results found in new.csv & old.csv.
 
+# TODO
+#  - make sure this works for comparing tags to facilitate release testing
+#  - handle the case when working directory is not the lintr directory
+
 suppressPackageStartupMessages({
   library(optparse)
+  library(data.table, include.only = "fwrite")
```

Comment:
I didn't know about this, nice to know 👍🏻

Reply:
OTOH: `fwrite()` is always namespaced below.
WDYT about `loadNamespace("data.table")` instead?
Will keep the search path clean while also erroring if the dependency is missing.

Reply:
I think `library()` is simpler for a dev script like this -- the search path concern is only relevant for interactive use which should only be used for debugging really. This script is designed for running in batch mode (hence the `#!/usr/local/bin/Rscript` at the top)